### ISSUE TITLE
[css-typed-om] Math functions that correspond css math functions declared in css-values-4

### ISSUE THREAD

**Issue by DevSDK** [w3c/css-houdini-drafts#1090]
Hi folks,

I noticed that [CSS Typed OM Level 1](https://drafts.css-houdini.org/css-typed-om-1) is now missing support for [CSS math functions that are part of the CSS Values and Units Module Level 4.](https://www.w3.org/TR/css-values-4/#calc-syntax)

I wonder if it was just missing or having some reason.

Thanks!

**Comment by tabatkins** [comment]
No, they're just missing because I haven't gotten to representing them. They can absolutely persist into any of the value stages that TypedOM can return.

**Comment by AtkinsSJ** [comment]
For now, what's the best approach for handling math functions that don't have corresponding CSSMathValue types?

Chrome's behaviour seems like a mix of throwing SyntaxErrors, and returning null. (Also sometimes crashing 😅)
